libpod/container_internal_linux.go

```diff
@@ -2116,6 +2140,39 @@ func (c *Container) copyOwnerAndPerms(source, dest string) error {
 	return nil
 }
 
+// chownSourceVolume changes the ownership of a source volume.
+func (c *Container) chownSourceVolume(path string, UID, GID int) error {
+	fi, err := os.Lstat(path)
+	if err != nil {
+		// Skip if path does not exist
+		if os.IsNotExist(err) {
+			logrus.Debugf("error returning file info of %q: %v", path, err)
```

Comment:
Message should mention chowning.

Reply:
This probably should be an error as well, since it would not be expected for this source to not exist.

Reply:
Finally the err will include the path, so dont put in in the message, otherwise the code will stutter.